third_party/fastutil.proguard

```diff
@@ -0,0 +1,8 @@
+-dontobfuscate
```

Comment:
@fmeum Unfortunately, our internal policy doesn't allow importing this file under bazel's own third_party to prevent potential bundling dependencies. I assume this is authored by you, right? Can you move this to the tools directory instead?

Reply:
Do you mean that I should move the file into `tools`, but keep the `genrule` referencing it in `third_party`? Wouldn't that cause the file to show up under `@bazel_tools//tools`?

Reply:
Yes, as long as it's not included in https://github.com/bazelbuild/bazel/blob/dd48c1a766519b5ad95ffdb515d6097ef3739124/tools/BUILD#L43, it should be fine.

Reply:
So maybe under `tools/proguard`?

Reply:
I moved it into `tools` and added it to `exclude` so that excluding it is documented as an explicit choice.

Reply:
SG, thank you!

Reply:
To fix the CI breakage, you probably have to sync to HEAD.